src/main/java/org/jabref/logic/citationstyle/CitationStyle.java

```diff
@@ -115,33 +118,51 @@ public static CitationStyle createCitationStyleFromFile(final String styleFile)
         return null;
     }
 
+    /**
+     * Provides the default citation style which is currently IEEE
+     * @return default citation style
+     */
     public static CitationStyle getDefault() {
         return createCitationStyleFromFile(DEFAULT);
     }
 
     /**
-     * THIS ONLY WORKS WHEN JabRef IS STARTED AS AN APPLICATION (JAR)
-     *
-     * Reads all available CitationStyle in the Jar
+     * Provides the citation styles that come with JabRef.
+     * @return list of available citation styles
      */
     public static List<CitationStyle> discoverCitationStyles() {
+        if (!STYLES.isEmpty()) {
+            return STYLES;
+        }
         try {
-            final List<CitationStyle> citationStyles = new ArrayList<>();
             String path = CitationStyle.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath();
 
+            // This is a quick fix to have the styles when running JabRef in a development environment.
+            // The styles.jar is not extracted into the JabRef.jar and therefore, we search the classpath for it.
+            if ((new File(path)).isDirectory()) {
```

Comment:
Whenever we add/modify file IO code in JabRef, we try to use the new Java API for that `java.nio`, not `java.io`. So please replace the condition with `Files.isDirectory(Paths.get(path))`.

Reply:
OK. No problem.